SwarmNet/src/kilobot.cpp

```diff
@@ -0,0 +1,173 @@
+#pragma once
+#include "../../../src/kilolib.h"
+#include "swarmnet.h"
+#include <math.h>
+
+
+class mykilobot : public kilobot
+{
+
+/*----------------------------------------------------------------------*/
+/*-----------------------------           ------------------------------*/
+/*----------------------------- User code ------------------------------*/
+/*-----------------------------           ------------------------------*/
+/*----------------------------------------------------------------------*/
+
+    message_t message;
+	Swarmnet s;
+    Channel *c;
+    Publisher p;

```

Comment:
I used abbreviations for the templet, which is bad
Use full names when defining the variables